lib/apivore/validator.rb

```diff
@@ -21,11 +21,9 @@ def matches?(swagger_checker)
       unless has_errors?
         send(
           method,
-          *RailsShim.action_dispatch_request_args(
             full_path(swagger_checker),
             params: params['_data'] || {},
             headers: params['_headers'] || {}
-          )
```

Comment:
@ran010  Do we not need this anymore? If not, please align the arguments with the `method` param.